.cloudtest/packet/create-kubernetes-cluster.sh

```diff
@@ -21,9 +21,22 @@ wait
 scp ${SSH_OPTS} ./.cloudtest/packet/start-master.sh root@${master_ip}:start-master.sh || exit 3
 scp ${SSH_OPTS} ./.cloudtest/packet/download-worker-images.sh root@${worker_ip}:download-worker-images.sh || exit 4
 
+pids=""
 ssh ${SSH_OPTS} root@${master_ip} ./start-master.sh &
+pids+=" $!"
+
 ssh ${SSH_OPTS} root@${worker_ip} ./download-worker-images.sh &
-wait
+pids+=" $!"
+
+for pid in $pids; do
+  echo "waiting for PID " "$pid"
+  wait "$pid"
+  exitcode=$?
+  if [ "$exitcode" != 0 ]; then
+    echo "node setup failed" && exit 9
```

Comment:
should we also throw `exitcode` instead of `9`?

Reply:
I think no, because there is own exit code for each line :)